configs/localhost-mac-system.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "Localhost",
+  "logo": "https://dytvr9ot2sszz.cloudfront.net/logz-docs/shipper-logos/local_host.png",
+  "description": "",
+  "hint": "Select the environment where you want to deploy Logz.io Telemetry Collector.",
+  "subtypes": [
+    {
+      "name": "Mac",
+      "logo": "https://dytvr9ot2sszz.cloudfront.net/logz-docs/shipper-logos/mac_machine.png",
+      "description": "Mac machine",
+      "hint": "Select your localhost OS.",
+      "datasources": [
+        {
+          "name": "System",
+          "logo": "https://dytvr9ot2sszz.cloudfront.net/logz-docs/shipper-logos/local_host.png",
+          "description": "Logz.io Uses OpenTelemetry to monitor your computer.",
+          "telemetries": [
+            {
+              "type": "LOG_ANALYTICS",
+              "hint": "Switch the toggle on if you want Logz.io Telemetry Collector to collect logs from your computer.",
+              "params": 
+              [
+                {
+                  "type": "path-list",
+                  "name": "logSources",
+                  "label": "Location of your logs",
+                  "required": true,
+                  "description": "The paths of log files to collect.",
+                  "hint": "By adding paths, OpenTelemetry will receive logs from these log paths and will export them to Logz.io.",
```

Comment:
OpenTelemetry will read logs from these log paths and will export them to Logz.io.